docs/document/content/features/readwrite-splitting/_index.cn.md

```diff
@@ -15,7 +15,7 @@ chapter = true
 
 与将数据根据分片键打散至各个数据节点的水平分片不同，读写分离则是根据SQL语义的分析，将读操作和写操作分别路由至主库与从库。
 
-![背景](https://shardingsphere.apache.org/document/current/img/readwrite-splitting/background.png)
+![背景] (https://shardingsphere.apache.org/document/current/img/readwrite-splitting/background.png)
```

Comment:
why add such blank space ?

Reply:
> hello, please review my pull request #15923

I think you got it wrong. Please search `SQL`, you can found it.